File: src/part3/code3-2.py
if __name__ == "__main__":
    N: int = int(input())
    v: int = int(input())

    a: list = []
    for _ in range(N): a.append(int(input()))

    found_id: int = -1
    for i in range(N):
        if a[i] == v:
            found_id = i
            break

    print(found_id)

File: src/part3/code3-2.cpp
#include <iostream>
#include <vector>
using namespace std;

int main() {
    // 入力を受け取る
    int N, v;
    cin >> N >> v;
    vector<int> a(N);
    for (int i = 0; i < N; ++i) cin >> a[i];
    
    // 線形探索
    int found_id = -1;
    for (int i = 0; i < N; ++i) {
        if (a[i] == v) {
            found_id = i;
            break;
        }
    }

    // 結果を出力
    cout << found_id << endl;
}
